src/org/scada_lts/mango/service/EventService.java

```diff
@@ -121,7 +123,9 @@ public void ackEvent(int eventId, long time, int userId, int alternateAckSource)
 	
 	@Override
 	public void insertUserEvents(int eventId, List<Integer> userIds, boolean alarm) {
-		userEventDAO.batchUpdate(eventId, userIds, alarm);
+		if(!isNewId(eventId)) {
+			userEventDAO.batchUpdate(eventId, userIds, alarm);
+		}
```

Comment:
In my opinion, this change of code may affect the change of program operation in the scope which was not the subject of the task